src/libraries/Microsoft.Extensions.Hosting.Systemd/src/SystemdHelpers.cs

```diff
@@ -13,6 +13,9 @@ namespace Microsoft.Extensions.Hosting.Systemd
     /// </summary>
     public static partial class SystemdHelpers
     {
+        private const string NOTIFY_SOCKET = "NOTIFY_SOCKET";
```

Comment:
Nit: This does not follow the repo naming conventions. https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md We use PascalCasing to name all our constant local variables and fields.

Reply:
Normally I would have just used a literal, but I was following the lead of [SystemNotifier](https://source.dot.net/#Microsoft.Extensions.Hosting.Systemd/SystemdNotifier.cs). Unlucky that I chose a bad example, I guess.